### DISCUSSION TITLE
Thoughts

### DISCUSSION THREAD

**Discussion by vickyyin1493** [vickyyin1493/Replication-FFJR#3] [General]
**Positive dividend firms**

1. They plan to use splits to signal dividend increases (signaling theory)
2. The market participants expect that and stock price continue to increase with positive returns in the year 

**Negative dividend firms**

1. They are having a negative return right now and they suspect that it’s due to the illiquidity of their stocks 
2. => they plan to use splits to increase liquidity (liquidity hypothesis)
3. (instead of signaling future dividends payout)
4. Market participants

- They expect dividend increase when splits are announced(the “norm” at the time that dividends are followed by splits) - (limited) data supports that but needs further literature on this (signaling theory)

- When they realised that there is no plan in dividend increase, their expectation fall and share price dropped => negative returns 

**Summary** 
In a way, this is like combining theory A’ and theory B’’.
Theory A′: Firms like to keep their stock price within certain bounds. When the stock price rises above a certain threshold, a firm may initiate a split, announcing it several weeks in advance of its effective date. Firms do not use splits to signal private information about firm prospects. Pre-split stock prices may be driven by information that suggests an imminent increase in dividends. 
Theory B′′: Firms use splits to signal private information about future dividend increases, which in turn signal private information about firm prospects. A firm will announce a split several weeks in advance of its effective date. - positive dividend

**Comment by shizhe75** [comment]
![image](https://github.com/vickyyin1493/Replication-FFJR/assets/135291810/7d04fd28-e9d1-4f01-8c20-01cb31c50358)

0.13 return filter

**Comment by shizhe75** [comment]
<img width="774" alt="image" src="https://github.com/vickyyin1493/Replication-FFJR/assets/135291810/c5afbbc0-854b-4e62-ae49-c1a0db18e356">

0.1301 return

**Comment by shizhe75** [comment]
Slight issue with the code - day_indexes uses data from "msi" which only goes from 1925 to 2009 which means there are very little data points in nyse_splits since we can match a lot of the announcement dates - may have to ask Ian about this if we can't get more contemporary data to match @vickyyin1493

**Comment by vickyyin1493** [comment]
An alternative theory could be:
Theory A′': Firms like to keep their stock price within certain bounds. When the stock price rises above a certain threshold, a firm may initiate a split, announcing it several weeks in advance of its effective date. Firms use splits to signal private information about future dividend increases, which in turn may signal private information about firm prospects.

**Comment by vickyyin1493** [comment]
Write-up:
1. introduce main theories, discuss with current literature 
2. what we did
3. results discussion - main + supplementary analysis on dividends (why certain theories don't make sense)
4. proposed alternative theory A''
